terraform/environments/oasys-national-reporting/locals_security_groups.tf

```diff
@@ -48,7 +48,6 @@ locals {
   }
 
   security_group_cidrs_by_environment = {
-    development   = local.security_group_cidrs_devtest
```

Comment:
only remove this when locals_development.tf is removed and the environment is destroyed